mkdocs.yml

```diff
@@ -47,12 +47,15 @@ nav:
           - 'SPI write protection': variants/asus_kgpe_d16/spi-wp.md
           - 'TPM measured boot': variants/asus_kgpe_d16/tpm-mboot.md
       - 'Talos II':
+          - 'Overview': variants/talos_2/overview.md
           - 'Releases': variants/talos_2/releases.md
           - 'Building manual': variants/talos_2/building-manual.md
-          - 'Installation manual': variants/talos_2/installation-manual.md
-          #- 'Hardware Configuration Matrix': variants/talos_2/hardware-matrix.md
+          - 'Initial deployment': variants/talos_2/initial-deployment.md
+          - 'Firmware update': variants/talos_2/firmware-update.md
+          - 'Recovery': variants/talos_2/recovery.md
+          - 'Hardware Configuration Matrix': variants/talos_2/hardware-matrix.md
           - 'Test matrix': variants/talos_2/test-matrix.md
-          - 'Conference materials': variants/talos_2/conferences.md
+          - 'Other manuals': variants/talos_2/other-manuals.md
```

Comment:
We should roll back the name of this file - `other manuals` is a collective term.

Reply:
renamed